src/lib.rs

```diff
@@ -77,6 +77,16 @@ impl Builder {
         self
     }
 
+    pub fn whitelisted_function<T: Into<String>>(mut self, arg: T) -> Builder {
```

Comment:
I think you just need to use `&T` there. Let me know if that works, otherwise... I guess this is good to merge, it's only an allocation at the beginning of the program anyway.